CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- [#218](https://github.com/qbic-pipelines/rnadeseq/pull/218) Release 2.2 to master
+- [#217](https://github.com/qbic-pipelines/rnadeseq/pull/217) Preparing release 2.2 with version bumps
```

Comment:
isn't the version bump #218?

Reply:
Oops, I messed up some numbers. Fixed! Thanks Júlia!